src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
+        String preparedDoughnut;
+
+        if (doughnutType.equals("Chocolate")) {
```

Comment:
Instead of checking one by one, you can save to a list and just iterate through. Even better would be a set which has faster access. This should get rid of this long list of doughnuts since it'll only get longer the more doughnuts we're selling